public/app/features/alerting/unified/components/contact-points/ContactPointHeader.tsx

```diff
@@ -67,9 +67,9 @@ export const ContactPointHeader = ({ contactPoint, onDelete }: ContactPointHeade
    * Used to determine whether to show the "Unused" badge
    */
   const isReferencedByAnything = usingK8sApi ? Boolean(numberOfPolicies || numberOfRules) : policies.length > 0;
-
   /** Does the current user have permissions to edit the contact point? */
   const hasAbilityToEdit = canEditEntity(contactPoint) || editAllowed;
+  debugger;
```

Comment:
👀

Reply:
What are you talking about, I don't see anything 🤫

Reply:
![https://media1.tenor.com/m/rRmnnnCnRGEAAAAd/sauron-eye.gif](https://media1.tenor.com/m/rRmnnnCnRGEAAAAd/sauron-eye.gif)